frontend/env.ts

```diff
@@ -0,0 +1,3 @@
+export const REDIRECT_URI = process.env.NEXT_PUBLIC_REDIRECT_URI
```

Comment:
I would rather not include the name of the framework (neither `PUBLIC`) in the environment variable, as framework can change in the future and this is independent to that.

I would also suggest changing `REDIRECT_URI` to `SPOTIFY_REDIRECT_URI` for consistency.

Reply:
I agree but after checking the Nextjs docs I've seen that it needs for the browser variables. I can check if I call the method using server-side rendering.

Reply:
This is actually a vulnerability, as using the `NEXT_PUBLIC` prefix for the secret and the client id will expose that to the browser ([docs](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)).